types/isomorphic-git__lightning-fs/isomorphic-git__lightning-fs-tests.ts

```diff
@@ -4,8 +4,8 @@ const FS = new LightningFS('FS');
 
 const originalData = new Uint8Array([138, 132]);
 
-FS.writeFile('/sample.txt', originalData).then(() => {
-  FS.readFile('./sample.txt').then((data: Uint8Array) => {
+FS.writeFile('/sample.txt', originalData, undefined, () => {
+  FS.readFile('./sample.txt', undefined, (_err: Error, data: string | Uint8Array) => {
```

Comment:
Seems weird that this is different than the [node.js type](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d7d9c1d76dead09380e702d858b40432a29d8a48/types/node/fs.d.ts#L2424) (the second arg should be optional), but the upstream types also are defined this way: https://github.com/isomorphic-git/lightning-fs/blob/6f3ae38eefdd2e2b0135e90b57e44bf29ecdb5bb/index.d.ts#L50 - so this is probably not worth over-defining on this side.